### PR TITLE
[HWKMETRICS-761] Log REST endpoint details when there is an error

### PR DESCRIPTION
**Pull request by rubenvp8510**
Do not merge yet.

### CODE REVIEW

**Comment by rubenvp8510** [comment]
This has an error when tried to read the request body.

File: api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/log/HttpErrorLoggerExtension.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+        private HttpHandler next;
+        Pattern postQueryPattern;
+
+        public HttpErrorHandler(HttpHandler next, Pattern postQueryPattern) {
+            this.next = next;
+            this.postQueryPattern = postQueryPattern;
+        }
+
+        private void getRequestBody(HttpServerExchange exchange) {
```
Comment (burmanm) [inline api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/log/HttpErrorLoggerExtension.java]: This method allocates huge amount of stuff, once the bytes from the stream to the memory, then a byte array and then a String from that byte array.

Comment (rubenvp8510) [reply api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/log/HttpErrorLoggerExtension.java]: Yeah I know, indeed this way of reading the request body doesn't work very well :(

**Comment by jsanda** [comment]
@rubenvp8510 can you please lower the logging level to WARN.

**Comment by rubenvp8510** [comment]
@jsanda Yes, I already set the level to WARN.

**Pull request merged**